docs/source/overview/quicktour.md

```diff
@@ -223,9 +119,25 @@ python -m olive.workflows.run --config config.json
     }
 }
 ```
+It is composed with 5 parts:
+### [Input Model](../tutorials/options.md/#input-model-information)
+You provide input model location and type. PyTorchModel, ONNXModel, OpenVINOModel and SNPEModel are supported model types.
+
+### [Host and Target Systems](../tutorials/options.md/#systems-information)
+An optimization technique, which we call a Pass, can be run on a variety of host systems and the resulting model evaluated on desired target systems. More details for the available systems can be found at at [OliveSystems api reference](systems).
+
+### [Evaluator](../tutorials/options.md/#evaluators-information)
+In order to chose the set of Pass configuration parameters that lead to the “best” model, Olive requires an evaluator that returns metrics values for each output model.
+
```

Comment:
You list your performance requirements (for example, accuracy, latency), that optimized candidate models should meet, here.

Reply:
added